scripts/importer/data_importer/raw_data_importer.py

```diff
@@ -325,9 +325,8 @@ def _insert_variants(self):
                         annotations = [dict(zip(vep_field_names, x.split('|'))) for x in consequence_array if len(vep_field_names) == len(x.split('|'))]
 
                     alt_alleles = base['alt'].split(",")
-                    if base['rsid'].startswith('rs'):
-                        rsids = [int(rsid.strip('rs')) for rsid in base['rsid'].split(';')]
-                    else:
+                    rsids = [int(rsid.strip('rs')) for rsid in base['rsid'].split(';') if rsid.startswith('rs')]
+                    if not rsids:
                         rsids = [None]
```

Comment:
I realise that this is not your code, but isn't it better to have `rsids` as an empty list instead of a list with one `None` element.

Reply:
Either keep it or add another case in:
```
if len(rsids) <= i:
    data['rsid'] = rsids[-1]  # same id as the last alternate
else:
    data['rsid'] = rsids[i]
```